<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1044</width>
    <height>678</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>391</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="layoutMid1">
     <item>
      <widget class="QPushButton" name="btnAjouter">
       <property name="text">
        <string>Ajouter joueur</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnModifier">
       <property name="text">
        <string>Modifier Joueur</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>520</y>
      <width>361</width>
      <height>78</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="layoutMid3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="btnAnnulerModif">
         <property name="text">
          <string>Annuler modifications</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnCompleterModif">
         <property name="text">
          <string>Compléter modifications</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="btnEffacer">
       <property name="text">
        <string>Effacer joueur</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="boxRecherche">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>570</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>241</width>
      <height>551</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>100</y>
      <width>391</width>
      <height>421</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="layoutMid2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Joueur</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="boxJoueur"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Âge</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="boxAge"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Personnage</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="boxPersonnage"/>
       </item>
       <item>
        <widget class="QCheckBox" name="boxPresent">
         <property name="text">
          <string>Présent</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Classe</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="boxClasse"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Race</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="boxRace"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Groupe</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="boxGroupe"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Niveau</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="boxNiveau"/>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Expérience</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="boxExp"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Notes</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="boxNotes"/>
       </item>
       <item>
        <widget class="QPushButton" name="btnPresence">
         <property name="text">
          <string>Inscrire présence</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>30</y>
      <width>301</width>
      <height>561</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="layoutRight1">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_13">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Joueurs présents :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblJoueurPresent">
         <property name="text">
          <string>nbdeJoueurs</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_14">
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Érèbe</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblErebe">
         <property name="text">
          <string>nbErebe</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_15">
       <item>
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Fanatiques</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblFanatique">
         <property name="text">
          <string>nbFanatiques</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_16">
       <item>
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Régiment oublié</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblRegiment">
         <property name="text">
          <string>nbRegiment</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_17">
       <item>
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Armada</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblArmada">
         <property name="text">
          <string>nbArmada</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_18">
       <item>
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>Dracosire</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblDracosire">
         <property name="text">
          <string>nbDracosire</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_19">
       <item>
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>Niveau Moyen</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblMoyenne">
         <property name="text">
          <string>moyenne</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <item>
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>Niveau médian</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblMedian">
         <property name="text">
          <string>median</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_12">
       <item>
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>Âge moyen</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblAgeMoyen">
         <property name="text">
          <string>age</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1044</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
